README.md

```diff
@@ -302,6 +302,31 @@ const App = () => (
     <td colspan="3">The active dot smoothly slides to its new position, and the 'passive' dot smoothly fades away from the area it once occupied</td>
   </tr>
   <tr>
+   <tr>
+    <td colspan="4" align='center'>
+      <h3>Slides animations:</h3>
+    </td>
+  </tr>
+  <tr>
+    <td>slidesAnimation</td>
+    <td>Animate while transitioning between slides. This feature only works when a single slide is visible</td>
+    <td><code>'default'</code></td>
+    <td>string</td>
+  </tr>
+  <tr>
+    <td colspan="4" align='center'>
+      <h3>Options for slides animations:</h3>
+    </td>
+  </tr>
+  <tr>
+    <td><code>'default'</code></td>
+    <td colspan="3">The slides transition smoothly and seamlessly, operating in a standard manner without any additional effects or animations</td>
```

Comment:
I didn't quite understand what this animation does.
And the statement that there is no animation is wrong, as there is one
Maybe it would be better to explain it like 'The slides change each other by sliding from left to right and vice verse, operating in a standard manner so to say' or modify it and make it more clear